common/src/main/kotlin/com/freeletics/gradle/plugin/FreeleticsMultiplatformExtension.kt

```diff
@@ -27,7 +27,7 @@ public abstract class FreeleticsMultiplatformExtension(project: Project) : Freel
         project.plugins.apply(FreeleticsAndroidBasePlugin::class.java)
 
         project.kotlinMultiplatform {
-            android {
+            androidTarget {
```

Comment:
Target was renamed https://kotlinlang.org/docs/whatsnew19.html#changes-to-android-target-support